### ISSUE TITLE
LR doesnt change on training or finetuning

### ISSUE THREAD

**Issue by FatihcanUslu** [OHF-Voice/piper1-gpl#124]
Hi,

I am trying to monitor how the learning rate changes when using LR schedulers. To do this, I added a LearningRateMonitor callback inside src/piper/train/__main__.py:

lr_monitor = LearningRateMonitor(logging_interval="step")

In TensorBoard, I can see the new lr-AdamW and lr-AdamW-1 entries, but their values remain constant (2e-4 and 1e-4) throughout training. They never update. Am I missing something?

I also noticed that the model uses self.automatic_optimization = False due to having multiple optimizers. Could this be preventing the LR schedulers from stepping and therefore causing the learning rate to stay constant?

Thank you.
